app/models/mixins/relationship_mixin.rb

```diff
@@ -615,6 +678,8 @@ def remove_parent(parent)
   end
 
   def remove_children(*child_objs)
+    return child_objs.each { |child| children.destroy(child.id) } if use_ancestry?
```

Comment:
I think we want to set the child's parent to nil?
does this actually delete the child record?

also, think this is loading all child objects another time and calling destroy on them - so it doesn't quite look right from that aspect either. But that is a tangent. Lets make sure we are disassociating the children from the parent rather than destroying the actual children.